### PR TITLE
--web added to device-install(.sh/.bat)

### PR DESCRIPTION
**Pull request by gjelsoe**
Solves #5382

### CODE REVIEW

**Review by fifieldt — APPROVED** [review]

File: bin/device-install.bat [hunk 1]
```diff
@@ -49,8 +52,14 @@ IF EXIST %FILENAME% IF x%FILENAME:update=%==x%FILENAME% (
     ) else (
         %ESPTOOL_CMD% --baud 115200 write_flash 0x260000 bleota-s3.bin
     )
-    for %%f in (littlefs-*.bin) do (
-        %ESPTOOL_CMD% --baud 115200 write_flash 0x300000 %%f
+    IF %WEB_APP%==1 (
+        for %%f in (littlefsweb-*.bin) do (
```
Comment (GUVWAF) [inline bin/device-install.bat:56]: Shouldn't this be `littlefswebui-`? Same for the .sh script.

Comment (gjelsoe) [reply bin/device-install.bat:56]: You are right, though I did changed it.

Comment (gjelsoe) [reply bin/device-install.bat:56]: It should be correct now, is a new pr needed?

Comment (gjelsoe) [reply bin/device-install.bat:56]: Np

**Pull request merged**